packages/jsii-pacmak/lib/targets/go/runtime/method-call.ts

```diff
@@ -17,6 +21,7 @@ export class MethodCall {
 
   private emitDynamic(code: CodeMaker) {
     code.line(`var ${this.returnVarName} ${this.concreteReturnType}`);
+    code.line(`${this.implMapVar} := make(${JSII_IMPL_MAP_TYPE})`);
```

Comment:
You don't seem to ever add things to this map (same below on L48)... And it really just gets passed to a method...

This makes me wonder:
- Do we need the variable, in fact?
- Can we not use an "empty map literal" instead of using `make` here?